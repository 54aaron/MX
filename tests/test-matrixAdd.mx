int main(){
    Matrix n;
    Matrix b;
    Matrix a;
    n = [[1,2,4],[3,4,5]];
    a = [[1,2,4],[3,4,5]];
    b = a +. n;
    prints("==============================================");
    prints("n = ");
    print_matrix(n);
    prints("==============================================");
    prints("a = ");
    print_matrix(a);
    prints("==============================================");
    prints("a +. n = ");
    print_matrix(b);
    prints("==============================================");
}
