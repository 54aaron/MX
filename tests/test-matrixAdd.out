==============================================
n = 

[ 1, 2, 4 ]
[ 3, 4, 5 ]
==============================================
a = 

[ 1, 2, 4 ]
[ 3, 4, 5 ]
==============================================
a +. n = 

[ 2, 4, 8 ]
[ 6, 8, 10 ]
==============================================
